### DISCUSSION TITLE
Questions about Kundur system

### DISCUSSION THREAD

**Discussion by MuaYoi** [CURENT/andes#491] [Q&A] [answered]
I feel very surprised about your hard work, but I also have some doubts and hope to receive your answer.
Q1: I am conducting some research using the kundur system which you provided. I find that all rows have the "trans" column of zero, even "Vn1" is not equal to "Vn2".
Q2: The input parameter values in this system seem to be inconsistent with the IEEE standard system. Is this your deliberate design, or can I find the xlsx file that uses the standard parameters?

I hope to receive your answers.
Best wishes for the success of your business.

**Comment by cuihantao** [comment, marked as answer]
Thank you!

The `trans` column is not used, but the handling of transformers are correct. The per unit impedances are assumed to be provided using `Vn1` as the base. 

There might be minor differences from the standard IEEE test systems. The test case was convertered from the raw file, which was created by J. Conto; it was used because it was available at the time. I haven't made the effort to match them exactly to IEEE systems, and you are welcome to contribute.

**Reply by MuaYoi** [reply]
Thank you very much, but I have one more question that why the phase angle θ of the slack generator change in TDS.

**Reply by cuihantao** [reply]
Slack generator is only valid in power flow. When you replace it with a dynamic generator whose M is not infinite, the generator speed changes so the bus voltage angle changes. Also check related books on power system analysis.

**Reply by MuaYoi** [reply]
I get it! Thanks for your reply and wish everything goes well with your work!
